docs/settings.rst

```diff
@@ -0,0 +1,35 @@
+.. _settings:
+
+Settings
+========
+
+pulp_rpm adds configuration options to the those offered by pulpcore.
+
+ALLOW_UNSAFE_ADVISORY_CONFLICT_RESOLUTION
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   This setting controls whether or not pulp_rpm will block advisory sync or
+   upload if it appears an 'incoming' advisory is incompatible with an existing
+   advisory with the same name.
+
+   This defaults to False, and advisory-merge will raise an ``AdvisoryConflict``
+   exception in two scenarios:
+
+   Situation 1
+      Updated date and version are the same but pkglists differ (and one is not a subset
+      or superset of the other).  E.g. It's likely a mistake in one of the pkglists.
+
+   Situation 2
+      Updated dates are different but pkglists have no intersection at all. E.g. It's
+      either an attempt to combine content from incompatible repos (RHEL6-main and RHEL7
+      debuginfo), or someone created a completely different advisory with already used id.
+
+
+   If this setting is True, Pulp will merge the advisories in Situation 1, and simply
+   accept the new advisory in Situation 2.
+
+.. note::
```

Comment:
can we mention that otherwise there should be manual intervention on the conflict resolution?

Reply:
Mmm, good point - there's fuller discussion in the issue, I'll work it into the workflow here.